utils/text/sanitizers.py

```diff
@@ -0,0 +1,32 @@
+# Python Standard Library Imports
+import html
+import re
+
+# isort: off
+
+
+def sanitize_cookie_value(value: str) -> str:
+    """Sanitize Cookie Value
+
+    Sanitizes a cookie value by escaping HTML special characters and
+    removing non-alphanumeric characters, except for some safe ones like
+    hyphens and underscores.
+
+    Args:
+    - value (str): The cookie value to be sanitized.
+
+    Returns:
+    - str: The sanitized cookie value.
+
+    References:
+    - https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie
+    - https://stackoverflow.com/a/1969339
+    """
+    # Escape HTML special characters to prevent XSS
+    sanitized_value = html.escape(value)
+
+    # Further restrict to a safe set of characters
+    sanitized_value = re.sub(
+        r'[^a-zA-Z0-9-_!#$%&\'()*+-./:<=>?@[\]^_`{|}~]', '', sanitized_value
```

Comment:
Hmm the S.O. comments further mention:

![image](https://github.com/hacktoolkit/django-htk/assets/422501/4e29c937-1082-4715-ac5b-41b866c45d19)

Seems like spaces _could_ be allowed. I won't make the change at this time, but just keep an eye on it.